tests/cli/test_auth.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import MagicMock, patch
+
+from click.testing import CliRunner
+from conjure_python_client import ConjureHTTPError
+from requests import HTTPError
+
+from nominal.cli.auth import set_token
+
+runner = CliRunner()
```

Comment:
probs slightly better to use a fixture for this?

Reply:
sure thing